.github/workflows/build.yml

```diff
@@ -0,0 +1,24 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
```

Comment:
Interesting to learn that the caching can be done behind the scenes instead of with the explicit Cache Action use and config. Thanks!